scripts/release_pypi.sh

```diff
@@ -53,7 +53,7 @@ fi
 
 echo "Generating PyPI source distribution..."
 cd "$GIT_ROOT"
-python3 -m build -s -w
```

Comment:
What's this change for?

Reply:
build by default already create wheels and source distribution, no need to pass in the arguments.

Reply:
Let's leave the arguments here? I think it's nicer, if only just as documentation of what we're doing.

Reply:
sg.